koku/masu/database/trino_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -731,7 +731,7 @@ SELECT azure.uuid as azure_uuid,
     max(subscription_name) as subscription_name,
     max(nullif(azure.resource_location, '')) as resource_location,
     max(azure.unit_of_measure) as unit_of_measure,
-    max(cast(azure.usage_quantity as decimal(24,9))) as usage_quantity,
+    max(cast(azure.quantity as decimal(24,9))) as quantity,
```

Comment:
Sorry I didnt put a comment on all of these :smile: these shouldnt be changed. `usage_quantity` is the normalised value we should keep.